src/current/_includes/releases/v25.1/feature-highlights-kv.html

```diff
@@ -0,0 +1,43 @@
+<table>
+ <thead>
+  <tr>
+   <th class="center-align" colspan="1" rowspan="2">Feature</th>
+   <th class="center-align" colspan="5" rowspan="1">Availability</th>
+  </tr>
+  <tr>
+    <th colspan="1" rowspan="1">Ver.</th>
+    <th colspan="1" rowspan="1" style="white-space: nowrap;">Self-hosted</th>
+    <th colspan="1" rowspan="1">Advanced</th>
+    <th colspan="1" rowspan="1">Standard</th>
+    <th colspan="1" rowspan="1">Basic</th>
+   </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td>
+    <p class="feature-summary">Admission Control Available on SQL Writes</p>
+    <p class="feature-description">
+      Added support for replication admission control on SQL writes, allowing writes and background cluster operations to be paced to prevent overloading the cluster during node restarts or joins.
```

Comment:
'replication admission control' could be a link to this new section of the AC docs that offers more details for those interested

https://www.cockroachlabs.com/docs/v25.1/admission-control#replication-admission-control

LGTM overall